docs/send-data/opentelemetry-collector/data-source-and-configurations/additional-configurations-reference.md

```diff
@@ -0,0 +1,284 @@
+
+The following is the file structure used in our configuration directory:
+
+```txt
+.
+├── conf.d
+│   ├── common.yaml
+│   └── hostmetrics.yaml
+├── env
+│   └── token.env
+└── sumologic.yaml
+```
+When the collector is started, it loads the configuration in the following order:
+
+* `sumologic.yaml`. This is the default configuration file provided by Sumo Logic. It contains the default settings for the collector.
+* All configuration files from `conf.d`, sorted alphabetically - these files contain additional configuration settings that can be customized by customers. If there are any conflicts between the files, the last loaded configuration file will take precedence.
+
+:::note
+If a configuration is loaded later in the order, it will be merged with the previous configuration.
+:::
+
+For example, if two configuration files define the same key, the value from the later file will overwrite the value from the earlier file. If a list or map is defined in multiple configuration files, the lists or maps are merged, with values from the later configuration file taking precedence.
+
+Let's consider the following example configuration files:
+
+```yaml title="conf.d/0-base.yaml"
+extensions:
+  sumologic:
+    collector_description: "My OpenTelemetry Collector"
+    collector_fields:
+      cluster: "cluster-1"
+    some_list:
+      - element 1
+      - element 2
+```
+
+```yaml title="conf.d/1-override.yaml"
+extensions:
+  sumologic:
+    collector_fields:
+      zone: "eu"
+    some_list:
+      - element 3
+      - element 4
+```
+
+The effective configuration will look like the following:
+
+```yaml
+extensions:
+  sumologic:
+    collector_description: "My OpenTelemetry Collector"
+    collector_fields:
+      cluster: "cluster-1"
+      zone: "eu"
+    some_list:
+      - element 3
+      - element 4
+```
+
+Note that the list has been overridden and maps have been merged.
+
+## Best Practices
+
+### Configuration Location
+
+All Sumo Logic preconfigured components are stored in `sumologic.yaml` file. This file is managed by the installation script and should never be changed manually.
+
+* It is recommended to maintain the configuration of all reusable components in `conf.d/common.yaml`.
+* Specific configurations should be grouped by pipelines in meaningfully named files in the `conf.d` directory.
+
+For example, a file named `conf.d/mysql.yaml` can contain the MySQL receiver along with any processors that are intended to modify the collected data before sending it to Sumo Logic.
+
+### Custom Pipeline
+
+There are few processors provided in `sumologic.yaml` which are intended to be used in every pipeline.
+
+* The memory limiter processor is used to prevent out of memory situations on the collector. It should be always first on the processors list. For mroe information, refer to the [OpenTelemetry documentation](https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor/memorylimiterprocessor#memory-limiter-processor).
```

Comment:
```suggestion
* The memory limiter processor is used to prevent out of memory situations on the collector. It should be always first on the processors list. For more information, refer to the [OpenTelemetry documentation](https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor/memorylimiterprocessor#memory-limiter-processor).
```